include/nicegraf.h

```diff
@@ -1246,6 +1246,8 @@ typedef struct ngf_attachment_description {
   ngf_sample_count sample_count; /**< Number of samples per pixel in the associated image. */
   bool is_sampled; /**< Whether the image associated with this attachment is sampled from a shader
                       at any point. */
+  bool is_resolve; /**< Whether the image associated with this attachment is used as a resolve
```

Comment:
```suggestion
  bool is_resolve; /**< Whether the image associated with this attachment is used as an MSAA resolve
```

Reply:
did an interactive rebase to fix the comment